main/forms.py

```diff
@@ -59,6 +59,29 @@ class CostRecoveryForm(forms.Form):
     )
 
 
+class FundingForm(forms.ModelForm):  # type: ignore [type-arg]
+    """Form to create and edit funding instances."""
+
+    class Meta:
+        """Meta class for the form."""
+
+        model = models.Funding
+        fields = (
+            "project",
+            "source",
+            "funding_body",
+            "cost_centre",
+            "activity",
+            "analysis_code",
+            "expiry_date",
+            "budget",
+            "daily_rate",
+        )
```

Comment:
As we want to include all fields in `Funding`, we can simply do:

```suggestion
        fields = "__all__"
```

Reply:
removed and added